dev/com.ibm.ws.request.timing/src/com/ibm/ws/request/timing/probeExtensionImpl/SlowRequestProbeExtension.java

```diff
@@ -75,6 +78,16 @@ public void processExitEvent(Event event, RequestContext requestContext) {
 			//Clean up activity for this request will be performed in the corresponding slow request timer task.
 
 			/** Instead of this should we just return false from invokeForEventExit? */
+		} else {
+			if ( requestContext.getisSlow()  &&  tc.isWarningEnabled()  &&  TraceComponent.isAnyTracingEnabled()  &&  tc.isDebugEnabled() ) { //if requestContext is a slowRequestContext
+				double activeTime = (System.nanoTime() - requestContext.getRootEvent().getStartTime())/1000000.0; //Calculate the active time for the request
```

Comment:
the "event" is the root event in this case.  so you can just say `System.nanoTime() - event.getStartTime())/1000000.0)`